node-commons/src/main/java/org/apache/iotdb/commons/schema/tree/AbstractTreeVisitor.java

```diff
@@ -284,10 +293,41 @@ protected final String[] generateFullPathNodes() {
         nodeNames.add(iterator.next().node.getName());
       }
     }
-    nodeNames.add(nextMatchedNode.getName());
+    return new PartialPath(nodeNames.toArray(new String[0]));
+  }
+
+  /**
+   * Get partial path from root to node.
+   *
+   * @param node node must be concluded in ancestorStack or nextMatchedNode
+   * @return partial path from traverse start node to the specified node
+   */
+  protected final PartialPath getPartialPathFromRootToNode(N node) {
```

Comment:
Maybe ```getCurrentNodePartialPath``` is better.

Reply:
The word currentNode is so vague that it can be difficult for users to understand.

In fact, the interface returns the path from root to the node in the parameter, which could be nextMatchedNode or any of the ancestor nodes. So I don't think currentNode is a better representation.